src/v2/guide/instance.md

```diff
@@ -10,15 +10,14 @@ Cada aplicación de Vue se comienza creando una nueva **Instancia de Vue** con l
 
 ```js
 var vm = new Vue({
-  // options
+  // opciones
 })
 ```
 
-Aunque no extrictamente asociado con el [patrón MVVM]
-(https://en.wikipedia.org/wiki/Model_View_ViewModel), el diseño de Vue fue en parte inspirado por él.
+Aunque no extrictamente asociado con el [patrón MVVM](https://en.wikipedia.org/wiki/Model_View_ViewModel), el diseño de Vue fue en parte inspirado por él.
```

Comment:
There is a typo here, it should be:

```suggestion
Aunque no estrictamente asociado con el [patrón MVVM](https://en.wikipedia.org/wiki/Model_View_ViewModel), el diseño de Vue fue en parte inspirado por él.
```

Reply:
Thanks for the suggestion, this was already in the code and I didn't saw it when I fixed the markdown issue. But the missing accent marks are completely my fault haha ;)